common/chat.ts

```diff
@@ -0,0 +1,60 @@
+export interface ChatMessage {
+  id: string
+  user: string
+  sent: number
+  data: {
+    type: string
+    text: string
+  }
+}
+
+// TODO(2Pac): Make this into an interface and include more information here
+export type ChatUser = string
```

Comment:
Seems like you intended to use this in actions but then never did? (I'd just delete tbh, I don't find it particularly useful for refactoring). Either delete or actually use it in these types, I have no particular preference as I don't think it makes refactoring any easier or harder